features/daily/call_permission.feature

```diff
@@ -0,0 +1,15 @@
+Feature: Call Permissions
+
+  Scenario: User call permission
+    Given there are telephony users with infos:
+      | firstname | lastname | exten | context | with_phone |
+      | Lagherta  | Unknown  | 1001  | default | yes        |
+      | Ragnar    | Lodbrok  | 1002  | default | yes        |
+      | Björn     | Ironside | 1003  | default | yes        |
+    Given there are call permissions with infos:
+      | name       | extensions | users            |
+      | permission | 1002       | Lagherta Unknown |
+    When "Lagherta Unknown" calls "1002"
+    Then "Ragnar Lodbrok" is hungup
```

Comment:
I have the feeling that this test can be a false negative. It would never fail, because the call have no time to be processed before looking if it's hungup/ringing
Do you think that if we wait 1-2 seconds to process the call and check if Lagherta is hungup, would be a better check?

Reply:
I added 2 seconds to allow Ragnar's phone to start ringing. Lagherta would need more wait time because she has to wait for the denied message to complete before being hungup.

Reply:
I think I would like to have these two steps (wait + hangup) to verify she is hangup